security/maltrail/src/opnsense/mvc/app/controllers/OPNsense/Maltrail/forms/sensor.xml

```diff
@@ -21,13 +21,13 @@
         <id>sensor.remoteserver</id>
         <label>Remote Server</label>
         <type>text</type>
-        <help>IP address of the remote logging server.</help>
+        <help>IP address of the remote logging server. This is necessary if you are using a centralized Maltrail instance, otherwise leave it blank to use that instance as the logging server.</help>
```

Comment:
```suggestion
        <help>IP address of the remote logging server. This is necessary if you are using a centralized Maltrail instance, otherwise leave it blank to use this instance as the logging server.</help>
```

?